playhouse/fields.py

```diff
@@ -26,12 +28,43 @@
 except ImportError:
     AES = Random = None
 
+try:
+    from bcrypt import hashpw, gensalt
+except ImportError:
+    hashpw, gensalt = None, None
+
 from peewee import *
 from peewee import binary_construct
 from peewee import Field
 from peewee import FieldDescriptor
 from peewee import SelectQuery
 
+
+class PasswordHash(bytes):
+            def check_password(self, password):
+                password = password.encode('utf-8')
+                return hashpw(password, self) == self
+
+
+class PasswordField(TextField):
```

Comment:
Rather than checking for `bcrypt` at initializiation time, I'd prefer to have a module-level `if` statement like I did for `AESEncryptedField`.